src/ui/components/SidePanel.tsx

```diff
@@ -37,8 +37,18 @@ function useInitialPrimaryPanel() {
 
   const { nags } = hooks.useGetUserInfo();
   const showTour = shouldShowTour(nags);
+  const { isAuthenticated } = useAuth0();
+
+  const { comments } = hooks.useGetComments(recordingId);
+
   const initialPrimaryPanel =
-    recording && isTestSuiteReplay(recording) ? "cypress" : showTour ? "tour" : "events";
+    recording && isTestSuiteReplay(recording)
+      ? "cypress"
+      : showTour
+      ? "tour"
+      : !isAuthenticated && comments.length > 0
+      ? "comments"
+      : "events";
```

Comment:
I think nested statements like this generally considered to be pretty hard to read. Something like this might be clearer:

```suggestion
let initialPrimaryPanel;
if (recording && isTestSuiteReplay(recording)) {
  initialPrimaryPanel = "cypress";
} else if (showTour) {
  initialPrimaryPanel = "tour";
} else if (!isAuthenticated && comments.length > 0) {
  initialPrimaryPanel = "comments";
} else {
  initialPrimaryPanel = "events";
}
```

Reply:
Thanks!